.pullapprove.yml

```diff
@@ -0,0 +1,9 @@
+approve_by_comment: true
+approve_regex: LGTM
```

Comment:
according to docs [here](http://docs.pullapprove.com/yaml/approve_regex/)
NOT LGTM